### ISSUE TITLE
Add the ability to display HVIF icons as image inside the document

### ISSUE THREAD

**Issue by Giova84** [Giova84/LittleWriter#13]
Currently, using the "Insert image" option, inside the document we can add the most common kind of images (png, jpeg and gif) however, would be also nice to have the possibility to diaplay HVIF icons: well, I know that probably these mime type wouldn't be displayed on other platforms when exported inside an editable document (HTML and ODT): anyway Little Writer is also able to print PDF documents, and here, since is a "printed" file, such images would be correctly displayed everywhere.

Currently, if you attempt to insert an HVIF icon inside a document, will be displayed as a generic icon: I suspect that this could be a limit of the Qt porting, since Little Writer relies on Qt.

**Comment by Paradoxianer** [comment]
The question here is if  there a haiku specific solution should be choosen.. if you use Translation kit instead of the QT solution you have HVIF support out of the box..

**Comment by Giova84** [comment]
I don't think that Qt is aware of Haiku's specific file formats/mime types; when I ported Scribus on Haiku, eg, the Scribus devs asked me to implement the same ability (import HVIF icons inside Scribus), but this can only be achieved using Haiku specific solution, eg rely on the Translation kit.

However this is a low priority ticket: Little Writer, currently needs more important things, like a better support for the ODF documents, see #5
